src/UIComponents/SelectTrainer.jsx

```diff
@@ -2,20 +2,28 @@
 import PropTypes from "prop-types";
 import React, { Component } from "react";
 import { connect } from "react-redux";
-import { setSelectedTrainer, getCompatibleTrainers } from "../redux";
+import { setSelectedTrainer, getCompatibleTrainers, setKValue } from "../redux";
 import { styles } from "../constants";
 
 class SelectTrainer extends Component {
   static propTypes = {
     selectedTrainer: PropTypes.string,
     setSelectedTrainer: PropTypes.func,
-    compatibleTrainers: PropTypes.object
+    compatibleTrainers: PropTypes.object,
+    setKValue: PropTypes.func, // set static propTypes for setKValue and kValue
+    kValue: PropTypes.number
   };
 
   handleChangeSelect = event => {
     this.props.setSelectedTrainer(event.target.value);
   };
 
+  /* add event handler -> handleChangeInput Function */
+  handleChangeInput = event => {
+    console.log("Console log for kValue:", event.target.value);
```

Comment:
And remove the debugging console log, here. :)

Reply:
done! 👍